packages/remix-react/routeModules.ts

```diff
@@ -43,6 +43,13 @@ export type CatchBoundaryComponent = ComponentType<{}>;
  */
 export type ErrorBoundaryComponent = ComponentType<{ error: Error }>;
 
+/**
+ * V2 version of the ErrorBoundary that eliminates the distinction between
+ * Error and Catch Boundaries and behaves like RR 6.4 errorElement and captures
+ * errors with useRouteError()
+ */
+export type V2_ErrorBoundaryComponent = ComponentType<{}>;
```

Comment:
We can get rid of the props object here

```suggestion
export type V2_ErrorBoundaryComponent = ComponentType;
```